### ISSUE TITLE
Mailing list

### ISSUE THREAD

**Issue by bkmgit** [inkstitch/inkstitch#1172]
Is there a mailing list for Inkstitch or can one be created?

**Comment by kaalleen** [comment]
No, sorry. I haven't heard that anyone has setup a mailing list yet. You are the first one asking ;)
